### PR TITLE
add: resume training feature

### PR DESCRIPTION
**Pull request by s0nicboOm**
Signed-off-by: kbatra <kushal_batra@intuit.com>

Introduce parameter `resume_training`. Based on this we decide whether we want to save optimizer dict in registry or not.

### CODE REVIEW

**Review by vigith — APPROVED** [review]
LGTM

**Review by ab93 — CHANGES_REQUESTED** [review]
Minor nitpicks

File: numalogic/models/autoencoder/pipeline.py [hunk 1]
```diff
@@ -59,6 +62,7 @@ def __init__(
         std_tolerance: float = 3.0,
         reconerr_method: str = "absolute",
         threshold_min: float = None,
+        resume_training: bool = False,
```
[the remark this answers is not in the document]
Comment (s0nicboOm) [reply numalogic/models/autoencoder/pipeline.py:65]: Cool

**Comment by s0nicboOm** [comment]
duplicate

**Pull request closed without merging**